vue-components/src/components/iconProps.ts

```diff
@@ -9,6 +9,6 @@ export enum IconTypes {
 	INFOOUTLINED = 'info-outlined',
 }
 
-export const iconColors = [ 'base', 'warning', 'error', 'notice', 'success' ];
+export const iconColors = [ 'base', 'warning', 'error', 'notice', 'success', 'current' ];
```

Comment:
Maybe using "inherit"? is there any place that we can look for inspiration?

Reply:
Yeah, I am split between `inherit` and `current`. `inherit` implies inheriting any style in CSS, but `currentColor` is the magic color in SVG that tells it to use the text color

Reply:
After thinking about it a bit more, I like `inherit` as well. I changed it to that :)

Reply:
What about calling it `chameleon` because it makes the icon change color based on its surroundings?
(kidding, `inherit` sounds good)